pkg/cmd/render/render_test.go

```diff
@@ -408,75 +408,69 @@ spec:
 			args: []string{
 				"--asset-input-dir=" + assetsInputDir,
 				"--templates-input-dir=" + templateDir,
 				"--asset-output-dir=",
 				"--config-output-file=",
 			},
-			testFunction: func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error {
-				issuerKeyFile := cfg.APIServerArguments["service-account-signing-key-file"]
-				// FIXME: Update when we wire a real default key
-				expectedIssuerKeyFile := kubecontrolplanev1.Arguments{"/etc/kubernetes/secrets/service-account.key"}
-				if !reflect.DeepEqual(issuerKeyFile, expectedIssuerKeyFile) {
-					return fmt.Errorf("expected the service-account-signing-key-file to be %q, but it was %q", expectedIssuerKeyFile, issuerKeyFile)
-				}
-				return nil
-			},
-		},
-		{
-			name: "scenario 9 user provided bound-sa-signing-key only no public part",
-			args: []string{
-				"--asset-input-dir=" + filepath.Join(assetsInputDir, "0"),
-				"--templates-input-dir=" + templateDir,
-				"--asset-output-dir=",
-				"--config-output-file=",
-			},
-			setupFunction: func() error {
-				data := `DUMMY DATA`
-				if err := os.Mkdir(filepath.Join(assetsInputDir, "0"), 0700); err != nil {
-					return err
-				}
-				return ioutil.WriteFile(filepath.Join(assetsInputDir, "0", "bound-service-account-signing-key.key"), []byte(data), 0644)
-			},
-			testFunction: func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error {
-				issuerKeyFile := cfg.APIServerArguments["service-account-signing-key-file"]
-				// FIXME: Update when we wire a real default key
-				expectedIssuerKeyFile := kubecontrolplanev1.Arguments{"/etc/kubernetes/secrets/service-account.key"}
-				if !reflect.DeepEqual(issuerKeyFile, expectedIssuerKeyFile) {
-					return fmt.Errorf("expected the service-account-signing-key-file to be %q, but it was %q", expectedIssuerKeyFile, issuerKeyFile)
-				}
-				return nil
-			},
-		},
-		{
-			name: "scenario 10 user provided bound-sa-signing-key only public part",
-			args: []string{
-				"--asset-input-dir=" + filepath.Join(assetsInputDir, "1"),
-				"--templates-input-dir=" + templateDir,
-				"--asset-output-dir=",
-				"--config-output-file=",
-			},
-			setupFunction: func() error {
-				data := `DUMMY DATA`
-				if err := os.Mkdir(filepath.Join(assetsInputDir, "1"), 0700); err != nil {
-					return err
-				}
-				return ioutil.WriteFile(filepath.Join(assetsInputDir, "1", "bound-service-account-signing-key.pub"), []byte(data), 0644)
-			},
-			testFunction: func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error {
-				issuerKeyFile := cfg.APIServerArguments["service-account-signing-key-file"]
-				// FIXME: Update when we wire a real default key
-				expectedIssuerKeyFile := kubecontrolplanev1.Arguments{"/etc/kubernetes/secrets/service-account.key"}
-				if !reflect.DeepEqual(issuerKeyFile, expectedIssuerKeyFile) {
-					return fmt.Errorf("expected the service-account-signing-key-file to be %q, but it was %q", expectedIssuerKeyFile, issuerKeyFile)
-				}
-				return nil
-			},
 		},
+		//
```

Comment:
Merging commented-out code seems like an anti-pattern to me, are you intending to address before merge or track resolution with bz etc?

Reply:
I think I wanted to make this mergable before the Holidays, but since I failed, I might just give it a try and address it before merge.